generator/internal/descriptor_utils.cc

```diff
@@ -652,15 +652,37 @@ std::string GetEffectiveServiceName(VarsDictionary const& vars,
   return std::string(name.data(), name.size());
 }
 
+/// If a service name mapping exists, return the replacement comment.
+/// If it does not exist, return a null optional.
+/// Parses a command line argument in the form:
+/// {"service_name_to_comments": "service_a=comment_a,service_b=comment_b"}.
+absl::optional<std::string> GetReplacementComment(VarsDictionary const& vars,
+                                                  absl::string_view name) {
+  auto service_name_to_comments = vars.find("service_name_to_comments");
+  if (service_name_to_comments != vars.end()) {
```

Comment:
I would have short-circuit this with:

```suggestion
  if (service_name_to_comments == vars.end()) return absl::nullopt;
```

Reply:
Done.